### ISSUE TITLE
About scale factor

### ISSUE THREAD

**Issue by joinling** [LuckyTiger123/DropMessage#1]
Thanks for your work!
When reading your article, I found that the message matrix will be scaled after dropmessage, but I don't seem to find the specific code implementation. Can you help me point it out?

**Comment by LuckyTiger123** [comment]
Hi, thanks for your interests. We use torch.nn.functional.dropout during the message propagation. The function itself will scale the input. More datails about  torch.nn.functional.dropout can be found in https://pytorch.org/docs/stable/generated/torch.nn.Dropout.html#torch.nn.Dropout.

**Comment by joinling** [comment]
I see. Thanks for your answer！
